server/interfaces/interfaces.go

```diff
@@ -1214,6 +1214,9 @@ type ConfigSecretProvider interface {
 
 type AuditLogger interface {
 	Log(ctx context.Context, resource *alpb.ResourceID, action alpb.Action, request proto.Message)
+	LogWithGroup(ctx context.Context, groupID string, action alpb.Action, request proto.Message)
```

Comment:
Can you change `With` to `For`? These functions generate audit events /for/ a specific resource type, rather than /with/ a specific resource.

Reply:
Pushed a change for this